cordformation/src/test/resources/postgres.gradle

```diff
@@ -0,0 +1,120 @@
+ext {
+    postgresql_version     = '42.2.12'
+    postgres_image_version = '12'
+    dbUser                 = 'myuser'
+    dbPassword             = 'mypassword'
+    dbSchema               = 'myschema'
+    dbName                 = 'mydb'
+    dbPort                 = 5432
+    dbHostName             = 'localhost'
+    dbInit                 = 'Postgres_init.sh'
+    dbDockerfile           = 'Postgres_Dockerfile'
+    postgres = [
+            dataSourceProperties: [
+                    dataSourceClassName: 'org.postgresql.ds.PGSimpleDataSource',
+                    dataSource: [
+                            user    : dbUser,
+                            password: dbPassword,
+                            url     : "jdbc\":\"postgresql\":\"\"//\"\${DBHOSTNAME}\":\"\${DBPORT}/\${DBNAME}\"?\"currentSchema\"=\"\${DBSCHEMA}",
```

Comment:
I think this example JDBC URL template can be simplified slightly:
```
url : "\"jdbc:postgresql://\"\${DBHOSTNAME}\":\"\${DBPORT}\"/\"\${DBNAME}\"?currentSchema=\"\${DBSCHEMA}",
```
The "trick" is that Typesafe interprets quoted text literally, and so we only need to put quotes around the prefix text, the suffix text and the text between the substitutions. And the quotes must also balance, of course.

I am slightly concerned that we are exposing Typesafe's limitations in our API here.

Reply:
Replace with URL template now. One option would be to escape the [forbidden characters ](https://github.com/lightbend/config/blob/master/HOCON.md), but this may clutter the code

Reply:
> One option would be to escape the [forbidden characters ](https://github.com/lightbend/config/blob/master/HOCON.md), but this may clutter the code

For interest, did you explore rewriting the JDBC URL template to "manage" TypeSafe Config's limitations? This would be a "Nice To Have"...

Reply:
Added a function to add the required escape elements